gulp.config.js

```diff
@@ -41,34 +45,39 @@ module.exports = function () {
     var report = {
         path: 'report/'
     };
-    var browserSync = {
-        dev: {
-            port: 3000,
-            server: {
-                baseDir: './src/',
-                middleware: [historyApiFallback()],
-                routes: {
-                    "/node_modules": "node_modules",
-                    "/src": "src"
-                }
+
+    if (global.env === 'dev')
```

Comment:
C:\dev\angular2-starter>gulp build --exclude-source --prod
[16:44:39] Using gulpfile C:\dev\angular2-starter\gulpfile.js
[16:44:39] Starting 'build'...
[16:44:39] Starting 'clean-report'...
[16:44:39] Starting 'clean-ts'...
[16:44:39] Finished 'clean-report' after 6.99 ms
[16:44:39] Finished 'clean-ts' after 24 ms
[16:44:39] Starting 'tsc'...
[16:44:43] Finished 'tsc' after 4.74 s
[16:44:43] Starting 'unit-test'...
07 06 2016 16:44:49.221:INFO [karma]: Karma v0.13.22 server started at http://localhost:9876/
07 06 2016 16:44:49.229:INFO [launcher]: Starting browser PhantomJS
07 06 2016 16:44:50.816:INFO [PhantomJS 2.1.1 (Windows 8 0.0.0)]: Connected on socket /#T-pSbDyh595rPdjEAAAA with id 68686206
07 06 2016 16:44:53.273:WARN [web-server]: 404: /base/node_modules/karma-ie-launcher/test/launcher.spec
PhantomJS 2.1.1 (Windows 8 0.0.0) ERROR
  Error: XHR error (404 Not Found) loading http://localhost:9876/node_modules/karma-ie-launcher/test/launcher.spec
        Error loading http://localhost:9876/node_modules/karma-ie-launcher/test/launcher.spec


07 06 2016 16:44:53.337:WARN [web-server]: 404: /base/node_modules/karma-phantomjs-launcher/test/add.spec